pkgs/desktops/gnome/extensions/gjs-osk/default.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  jq,
+}:
+
+let
+  uuid = "gjsosk@vishram1123.com";
+in
+stdenv.mkDerivation rec {
+  pname = "gjs-osk";
+  version = "unstable-2024-09-24";
+
+  src = fetchFromGitHub {
+    owner = "Vishram1123";
+    repo = pname;
+    rev = "2506a863b277339805519bca06d1b101cf92eb32";
+    sha256 = "sha256-AogDo0A50SLY2YiMdtXTG/j1m3Zgu9CMTxJjNyWu4bE=";
+  };
+
+  nativeBuildInputs = [ jq ];
+
+  passthru = {
```

Comment:
We typically place `passthru` in the bottom, see https://github.com/jtojnar/nixpkgs-hammering/blob/922ec48e954235772735191acc9f2cefeb7b6604/explanations/attribute-ordering.md.